contract/src/adjudicators/Counter.sol

```diff
@@ -24,7 +24,8 @@ contract Counter is IAdjudicator {
     /// @notice Error thrown when insufficient signatures are provided
     error InsufficientSignatures();
 
-    uint256 private constant FINAL_COUNTER = 1000;
+    // The custody contract now determines if a state is final based on signatures
+    // not on reaching a specific counter value
     uint256 private constant HOST = 0;
     uint256 private constant GUEST = 1;
 
```

Comment:
CounterApp {
  uint256 value; // Participant agree on the starting value
  uint256 final; // Participant agree on the final count
  uint256 version; // Each version of CounterApp increment version number from 0  
}